.github/workflows/api.yml

```diff
@@ -3,9 +3,6 @@ name: API
 on:
   push:
 
-env:
-  NODE_SERVER: BUILDNET
-
```

Comment:
I think we should keep this and use the config file to set the `NODE_SERVER` we want to us.

Reply:
@thomas-senechal  IMO that's why we added i've added the yaml file where we set the default NODE_SERVER and DNS to use , for your use case, you will need to just update the config_network.yml file

Reply:
@thomas-senechal also NODE_SERVER  was used in `run: xvfb-run ./massastation --node-server $NODE_SERVER`but the point of this PR is to not use  --node-server  flag anymore ...

Reply:
I'm talking about using the config file in the CI, to allow us to switch network easily if needed

Reply:
can you explain more please ? the config_network.yml file is already used otherwise the ci won't work , everytime massa  station starts , it init the NODE_SERVER with the default value!

Reply:
will be covered in https://github.com/massalabs/station/issues/1079